2019-06-28-Nitrogen-grid-optimization/send_mols_to_server.py

```diff
@@ -40,6 +40,12 @@ def make_ptl_mol(oemol):
     coords = oemol.GetCoords()
     symbols_list = [oechem.OEGetAtomicSymbol(atom.GetAtomicNum()) for atom in mol.GetAtoms()]
 
+    #convert to bohr
+    print(coords)
+    for key, item in coords.items():
+        coords[key] = (item[0]*1.88973, item[1]*1.88973, item[2]*1.88973)
```

Comment:
In the future I do recommend `qcel.constants.conversion_factor('angstrom', 'Bohr')` to get much more accurate conversions. Since we have geometry optimizations here, it isn't too big of a deal.